src/server/util/API.js

```diff
@@ -66,11 +66,17 @@ async function fetchRepositories(organization, page) {
 }
 
 async function getRepositories(organization) {
-    console.log('Called')
     const results = []
-    for(var page =1; page<=3; page++)
+    let page = 1
+    while(true)
     {
-        results.push(await fetchRepositories(organization, page))
+        repositories = await fetchRepositories(organization, page)
+        if(repositories.length === 0)
```

Comment:
You can check for <= 99 as it will save additional network call. But make sure to move result.push before this if